Resources/Locale/en-US/ui/actionmenu.ftl

```diff
@@ -0,0 +1,23 @@
+# Action menu stuff (left panel, with hotbars etc)
+
+ui-actionmenu-function-lock-action-slots = (Un)lock dragging and clearing action slots
+ui-actionmenu-function-open-abilities-menu = Open action menu
+ui-actionmenu-function-hotbar1 = Hotbar slot 1
```

Comment:
the hotbar and loadout entries don't seem to be used anywhere, but I changed their names also